src/libsystemd/sd-json/json-util.c

```diff
@@ -92,6 +92,22 @@ int json_dispatch_user_group_name(const char *name, sd_json_variant *variant, sd
         return 0;
 }
 
+int json_dispatch_chaddr(const char *name, sd_json_variant *variant, sd_json_dispatch_flags_t flags, void *userdata) {
```

Comment:
Hmm, this is only useful in the libsystemd-network context. Hence move it there?

Reply:
I moved this into the lease file instead. Is that what you were thinking?

Reply:
On second thought, I made it static too.